consensus/core/src/synchronizer.rs

```diff
@@ -720,11 +722,16 @@ impl<C: NetworkClient, V: BlockVerifier, D: CoreThreadDispatcher> Synchronizer<C
                                     for serialized_block in blocks {
                                         let signed_block = bcs::from_bytes(&serialized_block).map_err(ConsensusError::MalformedBlock)?;
                                         block_verifier.verify(&signed_block).tap_err(|err|{
+                                            let hostname = if context.committee.is_valid_index(signed_block.author()) {
```

Comment:
This should use `authority_index` instead?

Reply:
Ah good catch! Fixed it here https://github.com/MystenLabs/sui/pull/19597